demo/T1_meas_notebook/T1.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
I don't think measurement notebooks should be tracked by GitHub, I would remove it.